src/wazuh_modules/syscollector/schemas/hwinfo_syncrhonization.fbs

```diff
@@ -0,0 +1,22 @@
+include "common_agentInfo.fbs";
+
+table syscollector_hwinfo
+{
+    attributes_type: string;
+    attributes: HwInfoAttributeMsg;
+    index: string;
+    timestamp:string;
+}
+
+table HwInfoAttributeMsg
```

Comment:
In all the schemas, the `attributes` field is a table and not an union, is this expected?

Reply:
Done =)